sql/2019/17_CDN/17_05.sql

```diff
@@ -0,0 +1,9 @@
+#standardSQL
+# 17_05: Percentage of responses with strict-transport-security header

```

Comment:
@andydavies @colinbendell double checking... is "% of responses with the header" the metric you're looking for? Alternatively, we could query "% of pages having at least one response with the header".

Also, any restrictions on whether the response is 1st vs 3rd party? For example, if a Facebook embed includes one of these headers, its adoption would be inflated.

Reply:
I'm also not seeing anything CDN specific in metrics 05-13. Should these be limited to requests served by a CDN? Should they be grouped by CDN? cc @andydavies @colinbendell

Reply:
🛎 ping @andydavies @colinbendell